services/inc/services_dynalib.h

```diff
@@ -50,8 +50,8 @@ DYNALIB_FN(17, services, LED_RGB_SetChangeHandler, void(led_update_handler_fn, v
 DYNALIB_FN(18, services, log_print_direct_, void(int, void*, const char*, ...)) // Deprecated
 DYNALIB_FN(19, services, LED_GetColor, uint32_t(uint32_t, void*))
 
-DYNALIB_FN(20, services, log_message, void(int, const char*, const char*, int, const char*, void*, const char*, ...))
-DYNALIB_FN(21, services, log_message_v, void(int, const char*, const char*, int, const char*, void*, const char*, va_list))
+DYNALIB_FN(20, services, log_message, void(int, const char*, LogAttributes*, void*, const char*, ...))
+DYNALIB_FN(21, services, log_message_v, void(int, const char*, LogAttributes*, void*, const char*, va_list))
```

Comment:
how stable is `va_list`? Does it have any provision for binary compatibility between compiler revisions?  My concern is breaking logging when upgrading system part1 to a newer version that has a different layout for va_list (say after we upgrade to a new compiler version.)